index.html

```diff
@@ -709,6 +710,11 @@ <h3>
           <a data-for="PerformanceResourceTiming">resource info</a>'s
           [=response body info/content type=].
         </p>
+        <p data-dfn-for="PerformanceResourceTiming">
+          The <dfn>contentEncoding</dfn> getter steps are to return <a>this</a>'s
+          <a data-for="PerformanceResourceTiming">resource info</a>'s
+          [=response body info/content encoding=].
```

Comment:
The Fetch spec also needs to be updated.
The Fetch spec side PR will be similar to https://github.com/whatwg/fetch/pull/1481.

Reply:
Filed https://github.com/whatwg/fetch/pull/1742